packages/test-utils/package.json

```diff
@@ -18,5 +20,10 @@
     "mongodb-memory-server-core": "^6.9.2",
     "supertest-light": "^1.0.3"
   },
+  "preconstruct": {
+    "entrypoints": [
+      "index.ts"
+    ]
+  },
```

Comment:
This is the default so it's not necessary

```suggestion
```

Reply:
Ah cool, that's what I thought. I was having trouble getting VS code to play nice and this was part of the experiment to make it fly. I shall remove 👍